src/main/java/com/car/app/controller/CarService.java

```diff
@@ -2,14 +2,22 @@
 
 import java.util.List;
 
+import javax.ejb.EJB;
+import javax.ejb.Stateful;
+import javax.ejb.Stateless;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import com.car.app.model.Car;
 
+@Stateless
 public class CarService implements ICarService {
 	private static Logger log = LoggerFactory.getLogger(CarService.class);
-	private CarDao carDao = new CarDao();
+	
+	@EJB
```

Comment:
Usually we use @Inject to inject a bean, using EJB is to cases where we have a remote bean or a circular dependency or cases where @Inject is not capable of handling the injection. It is not wrong to use EJB, just unusual.

Reply:
Ok, let's use the usual :D